src/main/java/org/apache/maven/shared/utils/logging/AnsiMessageBuilder.java

```diff
@@ -203,6 +207,8 @@ public AnsiMessageBuilder format( String pattern, Object... args )
     @Override
     public String toString()
     {
-        return ansi.toString();
+        String result = ansi.toString();
+        System.out.println( result );
```

Comment:
Why is this nessary? All output is written through SLF4J.

Reply:
It's a work-in-progress as I explained in the commit message. I want to create a few screenshots in different environments so we can easily see what the outcomes will look like.